core/src/autogluon/core/utils/version_utils.py

```diff
@@ -106,7 +84,7 @@ def show_versions():
     versions = _get_autogluon_versions()
     sorted_keys = sorted(versions.keys(), key=lambda x: x.lower())
 
-    maxlen = max(len(x) for x in versions)
+    maxlen = 0 if len(versions) == 0 else max(len(x) for x in versions)
```

Comment:
I think previously the CI having error for max() having no argument is a sign that versions is an empty list, and I'm not sure if that's a correct behavior. I wonder does that mean version files are not correctly populated with the installation? Have you verified the installation locally?

Reply:
I could not reproduce it across local, container and linux environments.